app/controllers/concerns/devise_parameter.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module DeviseParameter
+  protected
+
+  def update_allowed_parameters
```

Comment:
This method has not been used anywhere. May be you need to add following code in the `app/controllers/api/v1/application_controller.rb`

```
 before_action :update_allowed_parameters, if: :devise_controller?
```

Reply:
We do not use the devise_controller for API (from what I understood). 
Though you are right: there is a problem with this module because the code is duplicated (in the module **and** in the main app_controller)
![image](https://user-images.githubusercontent.com/77609814/122334010-356d4e80-cf63-11eb-866e-2d1ac6e3712d.png)

So I will refactor that~

Reply:
You can use Rails concern feature to fix duplication. Check this `Authorization` module in [Braive](https://github.com/nimblehq/braive-web/blob/development/app/controllers/concerns/authentication.rb) project. It is used in the API and Web part

Reply:
Ok, it should be fixed! :)